src/App/Fossa/Analyze/VulnerabilityReport.hs

```diff
@@ -0,0 +1,138 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module App.Fossa.Analyze.VulnerabilityReport (
```

Comment:
I would strongly prefer to rename this file and all variables to something like `log4jReport`. I think `VulnerabilityReport` might cause confusion in thinking we support a dedicated vulnerability report, when in actuality we're only supporting log4j.

Reply:
I second this - we have an active request for vulnerability reports in cli. Best not to conflate